packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -804,14 +811,28 @@ export class VerticalStackedBarChartBase extends React.Component<
         onClick: this._onClick.bind(this, singleChartData),
         role: 'img',
       };
+      const classNames = getClassNames(this.props.styles!, {
+        theme: this.props.theme!,
```

Comment:
I see the className const declaration is getting repeated.

Reply:
fixed